.github/workflows/python-integrate.yaml

```diff
@@ -36,10 +36,10 @@ jobs:
             proto:
               - 'proto/**'
 
-      - name: Install Python 3.9
+      - name: Install Python 3.7
         uses: actions/setup-python@v4
         with:
-          python-version: '3.9'
+          python-version: '3.7'
```

Comment:
If I see correctly unit tests now will be run only in context of 3.7. Can we make them run in all supported versions (we can rely on matrix CI setting if that's helpful)

Reply:
Turns out it's pretty easy to set up and very much relevant to our needs 👍